app/mailers/user_mailer.rb

```diff
@@ -305,7 +305,7 @@ def edit_work_notification(user, work)
     I18n.with_locale(Locale.find(@user.preference.preferred_locale).iso) do
       mail(
         to: user.email,
-        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Your story has been updated"
+        subject: "#{t 'user_mailer.edit_work_notification.subject', app_name: ArchiveConfig.APP_SHORT_NAME}"
```

Comment:
Since we're getting rid of the notification, should this method be removed as well? Or do we currently use it elsewhere?